libs/application/templates/general-fishing-license/src/forms/GeneralFishingLicenseForm/fishingLicenseFurtherInfoSection.ts

```diff
@@ -0,0 +1,134 @@
+import {
+  buildCustomField,
+  buildDateField,
+  buildDescriptionField,
+  buildFileUploadField,
+  buildMultiField,
+  buildSection,
+  getValueViaPath,
+} from '@island.is/application/core'
+import { FormValue } from '@island.is/application/types'
+import { fishingLicenseFurtherInformation } from '../../lib/messages'
+import { FishingLicenseEnum } from '../../types'
+import { ATTACHMENTS_FIELD_ID, DATE_FIELD_ID } from '../../utils/fields'
+import {
+  licenseHasAreaSelection,
+  licenseHasFileUploadField,
+  licenseHasRailNetAndRoeNetField,
+  FILE_UPLOAD_ACCEPT,
+} from '../../utils/licenses'
+
+// Condition that determines whether given license has file upload field
+const hasFileUpload = (formValue: FormValue) => {
+  const selectedLicenseType = getValueViaPath(
+    formValue,
+    'fishingLicense.license',
+    '',
+  ) as FishingLicenseEnum
+  return licenseHasFileUploadField(selectedLicenseType)
+}
+
+// Condition that determines whether given license has special fields
+const hasRailNetAndRoeNetField = (formValue: FormValue) => {
+  const selectedLicenseType = getValueViaPath(
+    formValue,
+    'fishingLicense.license',
+    '',
+  ) as FishingLicenseEnum
+  return licenseHasRailNetAndRoeNetField(selectedLicenseType)
+}
+
+// Condition that determines whether given license has special fields
+const hasAreaSelection = (formValue: FormValue) => {
+  const selectedLicenseType = getValueViaPath(
+    formValue,
+    'fishingLicense.license',
+    '',
+  ) as FishingLicenseEnum
+  return licenseHasAreaSelection(selectedLicenseType)
+}
+
+export const fishingLicenseFurtherInfoSection = buildSection({
+  id: 'fishingLicenseFurtherInformationSection',
+  title: fishingLicenseFurtherInformation.general.sectionTitle,
+  children: [
+    buildMultiField({
+      id: 'fishingLicenseFurtherInformation',
+      title: '',
+      description: '',
+      children: [
+        // Custom field to replace section title
+        // i.e. to make it dynamic
+        buildCustomField({
+          id: 'fishingLicenseFurtherInformation.customTitle',
+          title: '',
+          doesNotRequireAnswer: true,
```

Comment:
Screens that have answers present (screen that has input field that has inputs that write to answers) should not have this as true. Can you double check all the fields?